<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xToolsToolBoxUiInputMenu</class>
 <widget class="QWidget" name="xToolsToolBoxUiInputMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Append suffix</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Append prefix</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="xToolsAffixesComboBox" name="comboBoxPrefix"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Escape character</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>CRC end index</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="xToolsSpinBox" name="spinBoxEndIndex"/>
   </item>
   <item row="1" column="1">
    <widget class="xToolsAffixesComboBox" name="comboBoxSuffix"/>
   </item>
   <item row="6" column="1">
    <widget class="xToolsSpinBox" name="spinBoxStartIndex"/>
   </item>
   <item row="2" column="1">
    <widget class="xToolsEscapeCharacterComboBox" name="comboBoxEscape"/>
   </item>
   <item row="8" column="1">
    <widget class="xToolsCrcAlgorithmComboBox" name="comboBoxAglorithm"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>CRC algorithm</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>CRC start index</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="xToolsCheckBox" name="checkBoxAppendCrc">
       <property name="text">
        <string>Append CRC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="xToolsCheckBox" name="checkBoxBigEndian">
       <property name="text">
        <string>Big endian CRC</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xToolsEscapeCharacterComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsEscapeCharacterComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsAffixesComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsAffixesComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsCrcAlgorithmComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsCrcAlgorithmComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">xToolsCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">xToolsSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
